src/rest_api.py

```diff
@@ -27,7 +27,9 @@ def stack_recommender():
     if input_json and 'external_request_id' in input_json and input_json['external_request_id']:
         try:
             persist = request.args.get('persist', 'true') == 'true'
-            r = StackRecommender().execute(input_json, persist)
+            check_license = request.args.get(
+                'check_license', 'false') == 'true'
+            r = StackRecommender().execute(input_json, persist, check_license)
```

Comment:
@sara-02 I think you should also do the same changes in other API end-points 
https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/src/rest_api.py#L43
https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/src/rest_api.py#L63 these two API end-points is being called from StackAnalysis POST asynchronously https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/bayesian/api_v1.py#L940-L941